Dockerfile

```diff
@@ -251,6 +232,9 @@ RUN set -x \
 	&& cp bin/ctr /usr/local/bin/docker-containerd-ctr \
 	&& rm -rf "$GOPATH"
 
+# Used to detect running from within a container
+ENV FROM_DOCKERFILE=1
```

Comment:
I am inclined to remove `.dockerenv` when we get introspection, so I wouldnt use it here.